client/src/app/components/modify-members/modify-members.component.html

```diff
@@ -1,12 +1,14 @@
 <p>Inside {{view}} Component</p>
-<button mat-button (click)="obtainClients()">Obtain Clients</button>
-<ul *ngFor="let pers of users">
-    <li>
-        username = {{pers.user_name}},
-        firstname = {{pers.first_name}},
-        lastname = {{pers.last_name}}, 
-        role = {{pers.role}},
-        is_active = {{pers.is_active}},
-        e-mail = {{pers.email}}
-    </li>
-</ul>
\ No newline at end of file
+<form [formGroup]="checkoutForm" (ngSubmit)="onSubmit(checkoutForm.value)">
+
+  <div>
+    <label for="name">
+      FirstName
+    </label>
+    <input id="name" type="text" formControlName="name">
```

Comment:
Please add angular material for the Input, the label and the button. I believe all of those are already imported in the matmodule so it's quick. Just each time you work on the html add angular material stuff.

Reply:
Fixed in commit f746527c72e44fe8b5f91554b127d5315ab3fd52